### PR TITLE
Added more common typos

### PR DESCRIPTION
**Pull request by polluks**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7057,6 +7057,7 @@ configute->configure
 configuted->configured
 configutes->configures
 configutration->configuration
+confimation->confirmation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7060]:
```suggestion
confim->confirm
confimation->confirmation
confimations->confirmations
confimed->confirmed
confims->confirms
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -27221,6 +27222,7 @@ satistying->satisfying
 satric->satiric
 satrical->satirical
 satrically->satirically
+sattelit->satellite
 sattelite->satellite
 sattelites->satellites
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:27227]:
```suggestion
sattelit->satellite
sattelite->satellite
sattelites->satellites
sattelits->satellites
```

**Review by peternewman — CHANGES_REQUESTED** [review]
A few suggestions

**Review by luzpaz — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -28116,6 +28123,7 @@ sidde->side
 sideral->sidereal
 siduction->seduction
 sie->size, sigh, side,
+sience->science
```
Comment (FloEdelmann) [inline codespell_lib/data/dictionary.txt:28126]:
```suggestion
sience->science,silence,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -28123,7 +28123,7 @@ sidde->side
 sideral->sidereal
 siduction->seduction
 sie->size, sigh, side,
-sience->science
+sience->science,silence,
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28126]: whitespace missing after first comma

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -7057,6 +7057,11 @@ configute->configure
 configuted->configured
 configutes->configures
 configutration->configuration
+confim->confirm
+confimation->confirmation
+confimations->confirmations
+confimed->confirmed
+confims->confirms
 confimred->confirmed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7065]: Dictionary needs sorting:
```suggestion
confimred->confirmed
confims->confirms
```

**Pull request merged**